x-pack/test/api_integration/apis/endpoint/policy.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect/expect.js';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function({ getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const supertest = getService('supertest');
+  describe('Endpoint policy api', () => {
+    describe('GET /api/endpoint/policy-response', () => {
+      before(() => esArchiver.load('endpoint/policy'));
+
+      after(() => esArchiver.unload('endpoint/policy'));
+
+      it('should return one policy response for host', async () => {
+        const expectedHostId = '4f3b9858-a96d-49d8-a326-230d7763d767';
```

Comment:
Is this value in the `data.json.gz` file? Could we use the generator here instead and avoid having to add the `data.json.gz` file?

Reply:
https://github.com/elastic/endpoint-app-team/issues/396